components/delete_channel_modal/delete_channel_modal.test.tsx

```diff
@@ -84,7 +92,7 @@ describe('components/delete_channel_modal', () => {
             <DeleteChannelModal {...props}/>
         );
 
-        wrapper.find(Modal).props().onExited();
+        wrapper.find(Modal).props().onExited(document.createElement('div')); // Cannot invoke an object which is possibly 'undefined'
```

Comment:
The first error I was receiving here was that `onExited` requires an HTML argument. I noticed some other tests passing in `document.createElement('div')` which I'm assuming is just to provide some sort of HTML element for the test, so I ended up doing the same. That resolved that error.

However, as the comment mentions, I'm still receiving this error: `Cannot invoke an object which is possibly 'undefined'`.

Reply:
You shouldn't need to pass anything into `onExited`.
I think this issue is that `wrapper.find()` is returning a potentially `undefined` object since it might not find the object it's looking for. So you probably need to do some optional chaining or assertion on the result of `wrapper.find()`.

Reply:
I'm using optional chaining:

`wrapper.find(Modal).props().onExited?.();`

I'm seeing two errors:

1. Eslint: "Expected an assignment or function call and instead saw an expression." and one from typescript: "Expected 1 arguments, but got 0."  I don't think that one should matter for a test and can be ignored, but I'm not entirely sure.

2. Typescript error, that is what I mentioned before with `onExited` requiring an argument, specifically an HTMLElement.